### ISSUE TITLE
would it be possible to save playlist access?

### ISSUE THREAD

**Issue by MrPotatoZed** [frondeus/fvtt-syrin-control#44]
The new version is great and not having to use the player actually lets me put syrinscape to good use again since my players were horrible about downloading extra stuff. Thank you for pushing the people at foundry and syrinscape to work together at least a little bit. 

I am playing around with using the macro folders that are created quite easily with the 0.4.0-alpha-2 UI. However, having to go to the macro folder tab, open individual macros, then execute them seems more cumbersome than the previous dropdown menu approach. It may be that I'm just used to managing all my audio from one tab or that I'm not super fluent in macros but I'm having trouble finding a workflow for on the fly changes that is as smooth as the playlist dropdowns. It might also be that if there was any open space in my hotbar the macro approach would be smoother since then I could at least trigger macros with one click but there is rarely more than one open slot for macros. 

The macro approach seems like a fantastic feature as I can see how it would make setting up MATT tiles easier and it does allow for setting things up in a way that's not possible with the current playlist dropdowns.  Creating macros for all soundsets in a folder automatically is also very slick. However, from my perspective it seems better as a complement to the previous approach than as a replacement. There are plenty of times when I don't have time to set up tile triggers or ordered lists of macros and just want to manage audio on the fly. In that situation having everything available in one tab and accessible through easy to manage dropdowns where clicking on the selection triggered the audio was pretty great. There are also times when I won't use most of the soundsets from a folder such that the current approach ends up creating file/folder management work as I have to go back and delete stuff I won't use. This isn't the end of the world and I know alternative UI's mean extra work but if you're making the decision to get rid of the playlist interface because you believe the macro approach is superior I want to make sure these use cases are considered. 

Bottom line, if it's possible to bring back the previous playlist dropdowns as an alternative way to access soundsets it would be much appreciated. Thank you for all your work on this.

**Comment by frondeus** [comment]
Hi. First of all thanks for testing the pre-release version

Absolutely i will think about it how to do it correctly. I agree that executing macro manually is not as easy.

On the other hand there are some modules that helps with it.

For example, personally I'm using Quick Insert to quickly search for a macro and execute it on the fly. 

I know that "hey, use different module" is not a good solution therefore i need to handle playlists anyway :)

**Comment by MrPotatoZed** [comment]
I'm a big fan of quick insert but hadn't really been using it for macros. I'll try to use that to speed things up as a workaround. 

Thanks for thinking about how to do this.  No rush and I'm excited to see what cool stuff you come up with next.

**Comment by frondeus** [comment]
@MrPotatoZed I just released https://github.com/frondeus/fvtt-syrin-control/releases/tag/0.4.0-alpha-3
I hope it's more convenient than macros!

**Comment by MrPotatoZed** [comment]
Awesome! I'll do some testing and let you know if I see anything weird.
